pages/index.html

```diff
@@ -189,4 +189,4 @@ <h1 class="display-1">
     </script>
 </body>
 
-</html>
+</html>
```

Comment:
why was the trailing line removed here?

Reply:
VSCode automatically removed it for some reason, will revert